sw/device/lib/base/multibits.h

```diff
@@ -0,0 +1,22 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_BASE_MULTIBIT_H_
+#define OPENTITAN_SW_DEVICE_LIB_BASE_MULTIBIT_H_
+
+enum {
+  /**
+   * Enumeration for hardware mutlbit variables
```

Comment:
This should go on the `enum` itself, and should probably go into more detail about what these are, for SW folks without corresponding HW context.